dashboard/lib/policies/gender.rb

```diff
@@ -0,0 +1,26 @@
+class GenderPolicy
```

Comment:
Two thoughts:

1. Since we're establishing a new pattern, should we add file-level comments here or a local README referencing [the article](https://codeclimate.com/blog/7-ways-to-decompose-fat-activerecord-models/) that suggested policy objects?

2. Have you had any issues re: autoloading for files in dashboard/lib, particularly when running tests?  While I agree [this is where POROs _should_ go](https://medium.com/extreme-programming/what-goes-in-rails-lib-92c74dfd955e) in practice it's given me some trouble in the past.  I suspect I'm just doing something wrong... but would like to track it down.

Reply:
Oh, we should definitely add some higher-level documentation; I like the idea of having a local README; when we get more categories of helper objects, we could even have one readme per directory detailing the guiding philosophy behind that category of object.

And yeah, looks like the files aren't loading. Reading up on the autoloader logic here https://github.com/fxn/zeitwerk#file-structure it looks like the class names need to more closely mirror the file structure. So instead of `class GenderPolicy` I need to do `class Policies::Gender`. Which I actually kind of like better.

Reply:
Nice find, Elijah! I am very happy to see us move in this direction!

Reply:
I added a basic readme; PTAL to confirm it's not missing anything important (it's pretty minimal right now)